src/data/teamData.ts

```diff
@@ -0,0 +1,12 @@
+import Seamus_Brugh from "../Assets/team/Seamus Brugh.png";
```

Comment:
Do we want to store team photos in an S3 bucket? Or in our project repo?

Reply:
We can store them outside the repo. I was just getting the basic component built.

Reply:
Would you like me to setup the S3 or is that something that Dylan wants to do, or you?

Reply:
I created an S3 bucket on the AWS account Dylan set up, but need to read more about pricing!

Reply:
Should I merge this one and create another PR to add the images and modify the team card component when we decide the final option? or hold of until we have it setup?